packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -145,6 +145,7 @@ export function BlockSettingsDropdown( { clientIds, ...props } ) {
 									/>
 								) }
 							</MenuGroup>
+							{ children }
 							<BlockSettingsMenuControls.Slot
```

Comment:
I don't understand why are we adding another slot and can't use the BlockSettingsMenuControls API to add new items in this menu?

Reply:
@draganescu see my comment here: https://github.com/WordPress/gutenberg/pull/22427#discussion_r428552883